bootstrap/node_logic.go

```diff
@@ -125,16 +125,25 @@ func createConsensusAlgo(nodeConfig config.NodeConfig) func(ctx context.Context,
 	gossip services.Gossip,
 	blockStorage services.BlockStorage,
 	consensusContext services.ConsensusContext,
+	management services.Management,
 	signer signer.Signer,
 	parentLogger log.Logger,
 	metricFactory metric.Factory) consensusAlgo {
 
-	return func(ctx context.Context, gossip services.Gossip, blockStorage services.BlockStorage, consensusContext services.ConsensusContext, signer signer.Signer, parentLogger log.Logger, metricFactory metric.Factory) consensusAlgo {
+	return func(ctx context.Context, gossip services.Gossip, blockStorage services.BlockStorage, consensusContext services.ConsensusContext, management services.Management, signer signer.Signer, parentLogger log.Logger, metricFactory metric.Factory) consensusAlgo {
 		switch nodeConfig.ActiveConsensusAlgo() {
 		case consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX:
 			return leanhelixconsensus.NewLeanHelixConsensusAlgo(ctx, gossip, blockStorage, consensusContext, signer, parentLogger, nodeConfig, metricFactory)
 		case consensus.CONSENSUS_ALGO_TYPE_BENCHMARK_CONSENSUS:
-			return benchmarkconsensus.NewBenchmarkConsensusAlgo(ctx, gossip, blockStorage, consensusContext, signer, parentLogger, nodeConfig, metricFactory)
+			ref, err := management.GetCurrentReference(ctx, &services.GetCurrentReferenceInput{})
```

Comment:
consider refactoring this  out into a `members, err := getCurrentCommitteeMembers(management)`?

Reply:
added todo.